CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ accidentally triggering the load of a previous DB version.**
 * #1668 Cannot add dimension if hypertable has empty chunks
 * #1674 Fix time_bucket_gapfill's interaction with GROUP BY
 
+**Licensing changes**
+* Reorder and policies around reorder and drop chunks are now
+  accessible to community users
+* Gapfill functionality no longer warns about expired enterprise
```

Comment:
This is not correct because the license expiration warning does not check the license type

Reply:
Yes, because no other license can expire or should generate this message.